subsys/bluetooth/common/Kconfig

```diff
@@ -9,6 +9,7 @@ menu "Bluetooth buffer configuration"
 config BT_BUF_ACL_TX_SIZE
 	int "Maximum supported ACL size for outgoing data"
 	range 27 65535
+	default 37 if BT_MESH
```

Comment:
One minor thing. I was thinking that it should be `BT_MESH_GATT` since this configuration is only needed when GATT support is enabled in mesh.